main.c

```diff
@@ -0,0 +1,118 @@
+#include <stddef.h>
+#include <malloc.h>
+#include <stdbool.h>
+#include <stdio.h>
+
+typedef struct 
+{
+    int value;
+    struct Node *next;
+}Node;
+
+typedef struct 
+{
+    Node *head;
+}List;
+
+List* create()
+{
+    List *list = calloc(1,sizeof(List));
+    list->head = calloc(1,sizeof(Node));
+    return list;
+}
+
+int pop(List *list, int index)
+{  
+    if (list == NULL)
+    {
+        return;
```

Comment:
```suggestion
        return -1;
```
Отрицательное число сигнализирует об ошибке